assets/src/edit-story/components/panels/layer/panel.js

```diff
@@ -36,7 +36,10 @@ function LayerPanel() {
   return (
     <Panel
       name="layers"
-      initialHeight={numLayersVisible * LAYER_HEIGHT}
+      initialHeight={Math.min(
+        numLayersVisible * LAYER_HEIGHT,
+        window.innerHeight / 3
+      )}
```

Comment:
Why is this one here? I mean, it does make sense, but it doesn't really fit this PR?

Reply:
Since I work on a 13'' screen it's just seriously annoying having to resize this panel after every refresh to see anything else at all, especially with dev tools open.

Can change it back before finishing the PR though (or leave it as it is + create a new issue for it)

Reply:
I think #227 and #825 cover this already